int16 axis_count
float32[] axis_state

int16 button_count
bool[] button_state

int16 hat_count
bool[] hat_state
